src/minimetrics/core.py

```diff
@@ -292,7 +292,7 @@ def consider_force_flush(self):
             self._flush_event.set()
 
     def _emit(self, flushable_buckets: FlushableBuckets) -> Any:
-        if options.get("delightful_metrics.enable_envelope_forwarding"):
+        if options.get("delightful_metrics.enable_transport"):
```

Comment:
Let's leave the original name in place. Renaming options always sucks.

Reply:
Sure